torch/distributed/fsdp/_runtime_utils.py

```diff
@@ -190,6 +189,8 @@ def _check_flat_params_on_expected_device(state: _FSDPState, module: nn.Module):
     """
     cpu_device = torch.device("cpu")
     for handle in traversal_utils._get_fsdp_handles(module):
+        if not handle:
```

Comment:
I think we should get rid of this change since `_get_fsdp_handles()` filters the `None` handles.